spirv_glsl.cpp

```diff
@@ -8980,6 +9034,25 @@ void CompilerGLSL::emit_instruction(const Instruction &instruction)
 		GLSL_BFOP(unsupported_FUnordGreaterThanEqual);
 		break;
 
+	case OpReportIntersectionNV:
+		statement("reportIntersectionNV(", to_func_call_arg(ops[0]), ", ", to_func_call_arg(ops[1]), ");");
```

Comment:
Just use `to_expression` here. `to_func_call_arg` is used for OpFunctionCall is certain scenarios.

Reply:
Same with OpTraceNV and OpExecuteCallableNV.